packages/ui/src/components/ConfirmModal/IframeConfirmModal.tsx

```diff
@@ -38,6 +36,7 @@ const IframeConfirmModal = ({
         <div className={styles.content}>
           {isLoading && <LoadingIcon />}
           <iframe
+            sandbox={undefined}
```

Comment:
should we set a proper value for this? i think `allow-scripts` would be fine?

Reply:
as mentioned, setting undefined will enable all the restrictions including scripts not allowed. Should we intend to allow the scripts?

Reply:
sry I misunderstood- noob me